pkg/standalone/list.go

```diff
@@ -20,25 +20,27 @@ import (
 
 // ListOutput represents the application ID, application port and creation time.
 type ListOutput struct {
-	AppID    string `csv:"APP ID"`
-	HTTPPort int    `csv:"HTTP PORT"`
-	GRPCPort int    `csv:"GRPC PORT"`
-	AppPort  int    `csv:"APP PORT"`
-	Command  string `csv:"COMMAND"`
-	Age      string `csv:"AGE"`
-	Created  string `csv:"CREATED"`
-	PID      int
+	AppID          string `csv:"APP ID"`
+	HTTPPort       int    `csv:"HTTP PORT"`
+	GRPCPort       int    `csv:"GRPC PORT"`
+	AppPort        int    `csv:"APP PORT"`
+	MetricsEnabled bool   `csv:"-"`
```

Comment:
This is on purpose, so it does not pollute the stdout.

Reply:
We just need this property in the Dashboard.

Reply:
Add this explanation into the code (comment) itself? Thanks